charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,106 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team is the liaison between the Data Coordination Platform (DCP) and labs contributing data to the Human Cell Atlas (HCA). This team is:
+
+* Working with data contributors who are generating new data types to develop a deep understanding of their data
+* Assisting data contributors to describe their experiments in a manner that supports findability, reusability and reproducability within the DCP ecosystem
+* Helping data contributors to submit their data to the DCP, resolving any emerging issues during the process
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* -  the Ingest Support team member who assists labs contributing data to the HCA.
+
+*Data* - raw data files from an experiment such as [fastq files](https://en.wikipedia.org/wiki/FASTQ_format) for sequencing experiments or tiff stacks for imaging experiments
+
+*Metadata* - descriptive information about what projects, samples and experiments are and how they were collected or conducted
+
+*New data types* - assay data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
+
+*Established data types* - assay data types which have a history of submission and downstream use in the DCP and for which there are already many datasets of this type represented in the platform
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor to submit one at a time. For example, a data contributor might upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
+
+*The PM team* - the team responsible for the oversight of the DCP as the product, developing the shared understanding of the overall product vision, priorities, and release schedule with the community.
+
+## Objectives
+
+- Assisting data contributors uploading data and metadata to the DCP
+- Helping data contributors address errors with their data formatting and metadata before it can be submitted to the DCP. 
+- Ensuring a data contributor's experiment can be accurately represented in the DCP
+- Working with the data contributor to understand if new metadata fields are needed to represent their data
+- Proposing enhancements to the metadata schemas following the process defined by the Metadata Schema team
+- Working with the UX team to collect and provide feedback from data contributors to the Ingest Infrastructure team about how the infrastructure can be improved
```

Comment:
Although Tony describes his charter as _This PR is for the first draft of the ingest **infrastructure** charter_, the actual name he selected was Ingest Service.